main/main.cpp

```diff
@@ -1775,8 +1776,12 @@ bool Main::iteration() {
 
 	if (frame > 1000000) {
 
-		if (GLOBAL_DEF("debug/settings/stdout/print_fps", OS::get_singleton()->is_stdout_verbose())) {
-			print_line("FPS: " + itos(frames));
+		if (GLOBAL_DEF("debug/settings/stdout/print_editor_fps", OS::get_singleton()->is_stdout_verbose()) && editor) {
```

Comment:
I think printing FPS for the editor itself is pointless, as this method is inappropiate. Just check the Editor singleton for editor being enabled and skip printing FPS.

Reply:
Actually this is main.cpp so you should be able to do `if (GLOBAL_DEF("debug/settings/stdout/print_fps", OS::get_singleton()->is_stdout_verbose()) && !editor)`.

Reply:
Yeah I don't see an issue in using the editor bool directly as I did, but if there is let me know and I'll change it again.

Changed the commit, it keeps the current print_fps name so just the check to make sure that it's not running in the editor has been added.